docs/docs/features/studio.md

```diff
@@ -10,6 +10,37 @@ ZMK Studio is in beta. Although every effort has been made to provide a stable e
 
 ZMK Studio provides runtime update functionality to ZMK powered devices, allowing users to change their keymap layers without flashing new firmware to their keyboards. Studio is still under active development, and is not yet ready for casual end user use.
 
+## Capabilities
+
+ZMK Studio currently has the following capabilities:
+
+| Legend: | ✅ Supported | 🚧 Under Development | 💡 Planned | 🕯️ Low Priority | ❌ Not Planned |
+| :------ | :----------- | :------------------- | :--------- | :-------------- | -------------- |
+
+| Studio Status | **Feature or Capability**                                                                         |
+| :-----------: | ------------------------------------------------------------------------------------------------- |
+|      ✅       | Making keymap changes while keyboard is in use                                                    |
+|      ✅       | Making changes while connected via USB                                                            |
+|      ✅       | Native app for: Windows, Linux, MacOS                                                             |
+|      ✅       | Making changes while connected via BLE (Linux web-app & native apps only)                         |
+|      ✅       | Assigning [predefined behaviors](../keymaps/behaviors/index.mdx) to keys                          |
+|      ✅       | Assigning [user-defined behaviors](../keymaps/behaviors/index.mdx#user-defined-behaviors) to keys |
+|      💡       | [Configuring predefined behaviors](../config/behaviors.md)                                        |
+|      💡       | [Configuring user-defined behaviors](../config/behaviors.md)                                      |
+|      💡       | [Configuring tap-dance](../keymaps/behaviors/tap-dance.mdx)                                       |
+|      💡       | [Configuring macros](../keymaps/behaviors/macros.md)                                              |
+|      ❌       | Defining new behaviors not specified in devicetree                                                |
+|      💡       | Defining [combos](../keymaps/combos.md)                                                           |
+|      💡       | Defining [conditional layers](../keymaps/conditional-layers.md)                                   |
```

Comment:
Do the same caveats for behaviors (configuring existing/reserved vs. defining) apply here? This is a bit inconsistent if so.

Reply:
Yes, for sure, this'll require the same reserved approach when we implement it.

Reply:
I'm not sure it's worth making the distinction yet here, like in a previous point, but I'm happy to revise if you think it's needed.

Reply:
Honestly, if we are sticking to the current distinction these should be split to two ("configuring existing" -> planned, "defining" -> not planned).

But like I point out in the main review comment, I could see us going the other way and not making that distinction in this table at all and consider "configure" the same as "defining" and omitting the "reserved" detail.

Reply:
I've changed it to "configuring" but chosen to omit the not planned "defining" for the time being.